zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -577,32 +365,30 @@ mod tests {
 
         let expected_views = vec![
             // v_sapling_shard_unscanned_ranges
-            format!(
-                "CREATE VIEW v_sapling_shard_unscanned_ranges AS
-                SELECT
-                    shard.shard_index,
-                    shard.shard_index << 16 AS start_position,
-                    (shard.shard_index + 1) << 16 AS end_position_exclusive,
-                    IFNULL(prev_shard.subtree_end_height, {}) AS subtree_start_height,
-                    shard.subtree_end_height AS subtree_end_height,
-                    shard.contains_marked,
-                    scan_queue.block_range_start,
-                    scan_queue.block_range_end,
-                    scan_queue.priority
-                FROM sapling_tree_shards shard
-                LEFT OUTER JOIN sapling_tree_shards prev_shard
-                    ON shard.shard_index = prev_shard.shard_index + 1
-                INNER JOIN scan_queue ON 
-                    (scan_queue.block_range_start BETWEEN subtree_start_height AND shard.subtree_end_height) OR
-                    ((scan_queue.block_range_end - 1) BETWEEN subtree_start_height AND shard.subtree_end_height) OR
-                    (
-                        scan_queue.block_range_start <= prev_shard.subtree_end_height
-                        AND (scan_queue.block_range_end - 1) >= shard.subtree_end_height
-                    )
-                WHERE scan_queue.priority > {}",
-                u32::from(db_data.params.activation_height(NetworkUpgrade::Sapling).unwrap()),
```

Comment:
Note for future maintainers (hi us!) - hard-coding activation heights into the database is fine for settled network upgrades, but for ones in development (e.g. NU6 for us) this can lead to a problem if an activation height is picked, then crates published (and used) with that height, and then we discover a bug that requires us to change the consensus branch ID and activation height. In this case the hard-coded height would need to be fixed via a migration.